pyproject.toml

```diff
@@ -46,10 +46,10 @@ full = [
     "Pillow>=8.0.0",
 ]
 crypto = [
-    "cryptography; python_version >= '3.7'",
-    "PyCryptodome; python_version == '3.6'",
+    "cryptography; python_version >= '3.8'",
```

Comment:
cryptography still works with  python 3.7, right? Just because we no longer support 3.7, it doesn't mean that we have to make it actively harder to use pypdf with 3.7.

Reply:
the specific code for python 3.7 has been removed.  pypdf should not work anymore on this release